README.md

```diff
@@ -211,6 +211,50 @@ catch (OperationFailedException e){
 ### Sample Application 
 For a more detailed example on how to use the library please take a look at our [Sample App](sample/).
 
+## Testing
+
+We use [ethereumjs/testrpc](https://github.com/trufflesuite/ganache-cli) and [Truffle framework](http://truffleframework.com/) unit tests.
+
+For the SDK tests target, pre-actions and post-actions scripts in the KinTestHost scheme will setup truffle and testrpc running for the duration of the test.
+
+### Requirements
+
+Node.js and NPM. You can install these using homebrew:
+
+```bash
+$ brew install node
+```
+Next, install specific npm packages using:
+
+```bash
+$ cd truffle
+$ npm install
+```
+
+Next, initialize and update git submodules.  This will include `truffle/kin-token`.
+
+```bash
+$ git submodule init && git submodule update
+```
+
+### How to run the tests
+* From command line
```

Comment:
add an extra line here. 
(see how it looks here: https://github.com/kinfoundation/kin-sdk-core-android/blob/android-testrpc-network/README.md )